### ISSUE TITLE
Warning about overwriting (newer) saves

### ISSUE THREAD

**Issue by RevQuixo** [slinga-homebrew/Save-Game-Copier#27]
Not sure this is even possible but now that we have comments working and timestamp is a part of those comments could we throw a soft warning if you are attempting to copy over a newer version of a save that has the same name on a target device?  Maybe this could/should be extended to any instance where you are copying over a save where the target device has a save there already.

**Comment by slinga-homebrew** [comment]
Yeah I probably should have a warning about overwriting existing. I did it for formatting, should be able to re-use some of my code for that.

**Comment by amerika13** [comment]
Yeah, this was specifically a scenario I tested when testing the latest version you put out. I wanted to see if I could overwrite a newer save on my internal with my older save from a backup. And I could. So a message stating that the save is newer or older and if the user would like to continue would be a great QoL add and save some people some headaches/heartache too haha.
